### ISSUE TITLE
Examples do not start reliable, need multipe starts

### ISSUE THREAD

**Issue by jespersmith** [Halodi/eve-ros2-examples#6]

**Comment by jespersmith** [comment]
Fixed in #5

**Comment by jespersmith** [comment]
The examples call publisher.publish very quickly after creating the publisher. If the publisher did not connect fast enough, the message gets lost and the examples hangs.

To fix, we used a transient_local qos with depth of 1, so if the publisher connects after publishing the first message, the subscriber still receives the first target in the trajectory

Example code
```
      // Create a latching QoS to make sure the first message arrives at the trajectory manager, even if the connection is not up when publish_trajectory is called the first time.
      // Note: If the trajectory manager starts after this node, it'll execute immediatly.
      rclcpp::QoS latching_qos(1);
      latching_qos.transient_local();


    // set up publisher to trajectory topic
    publisher_ = this->create_publisher<halodi_msgs::msg::WholeBodyTrajectory>("/eve/whole_body_trajectory", latching_qos);

```
